apps/native-component-list/webpack/webpack.prod.js

```diff
@@ -25,38 +25,49 @@ module.exports = merge(common, {
     vendor: ['react', 'react-native-web'],
     app: appEntry,
   },
-  devtool: 'hidden-source-map',
+  devtool: 'source-map',
   plugins: [
-    new CleanWebpackPlugin([locations.production.folder]),
-    new webpack.optimize.OccurrenceOrderPlugin(),
-    new webpack.optimize.ModuleConcatenationPlugin(),
-    new CopyWebpackPlugin([
-      {
-        from: locations.template.manifest,
-        to: locations.production.manifest,
-      },
-      {
-        from: locations.template.serveJson,
-        to: locations.production.serveJson,
-      },
-    ]),
+    /** Delete the build folder  */
+    new CleanWebpackPlugin([locations.production.folder], {
+      root: locations.root,
+      verbose: true,
+      dry: false,
+    }),
+    /** Remove unused import/exports  */
+    new WebpackDeepScopeAnalysisPlugin(),
+
     new MiniCssExtractPlugin({
       filename: 'static/css/[contenthash].css',
       chunkFilename: 'static/css/[contenthash].chunk.css',
     }),
+
+    /** GZIP files */
     new CompressionPlugin({
       test: /\.(js|css)$/,
       filename: '[path].gz[query]',
       algorithm: 'gzip',
       threshold: 1024,
-      minRatio: 0.8
+      minRatio: 0.8,
     }),
+    /** secondary compression for platforms that load .br  */
     new BrotliPlugin({
-			asset: '[path].br[query]',
-			test: /\.(js|css)$/,
-			threshold: 1024,
-			minRatio: 0.8
-		})
+      asset: '[path].br[query]',
+      test: /\.(js|css)$/,
+      threshold: 1024,
+      minRatio: 0.8,
+    }),
+
+    /** Copy the PWA manifest.json and the caching policy serve.json from the template folder to the build folder  */
+    new CopyWebpackPlugin([
```

Comment:
Any reason not to do this in the common config since we want a manifest there too?

Reply:
These are copying to the `web-build` folder which AFAIK isn't used for dev builds